Podfile

```diff
@@ -5,7 +5,7 @@ platform :ios, '7.0'
 # ignore all warnings from all pods
 inhibit_all_warnings!
 
-pod 'toxcore-ios', :podspec => 'https://raw.githubusercontent.com/Chuongv/toxcore-ios/mannol/toxcore-ios.podspec' 
+pod 'toxcore-ios', :podspec => 'https://raw.githubusercontent.com/Chuongv/toxcore-ios/0.1.7-new-av/toxcore-ios.podspec' 
```

Comment:
As I told you in email, I've updated libvpx to use https instead of http. I think that it would be better to keep libvpx in our git repo.

1. This way we will always know what version of libvpx was used to build framework.
2. Maybe we'll add some script to build framework from latest source when updating pod.